shell.nix

```diff
@@ -0,0 +1,37 @@
+let
+  moz_overlay = import (
+    builtins.fetchTarball https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz
+  );
+  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
+  beta = nixpkgs.rustChannelOf { channel = "beta"; };
```

Comment:
Do we need beta? I thought this worked on Stable?

Reply:
Fixed.